README.md

```diff
@@ -30,6 +30,7 @@ in protocol buffer or text format and dumps the result as JSON to
 
 Installing and building:
 
+    $ cd cmd/prom2json 
```

Comment:
With fully qualified paths, you should be able to `go get` from anywhere (provided your Go build environment is set up).